analytics/aggregate_test.go

```diff
@@ -35,10 +35,31 @@ func TestAggregate_Tags(t *testing.T) {
 			Tags:  []string{"", "   ", "tag2"},
 		},
 	}
+	recordsDot := []interface{}{
+		AnalyticsRecord{
+			OrgID: "ORG123",
+			APIID: "123",
+			Tags:  []string{"tag1", ""},
+		},
+		AnalyticsRecord{
+			OrgID: "ORG123",
+			APIID: "123",
+			Tags:  []string{"", "...", "tag2"},
+		},
+		AnalyticsRecord{
+			OrgID: "ORG123",
+			APIID: "123",
+			Tags:  []string{"", " ...", "tag2"},
```

Comment:
I think it could be good to add some more "real-world" tags, something like `internal.group1.dc1.`